djangocms_page_meta/cms_toolbars.py

```diff
@@ -8,11 +8,15 @@
 from cms.toolbar_pool import toolbar_pool
 from cms.utils.i18n import get_language_list, get_language_object
 from cms.utils.permissions import has_page_permission
-from django.core.urlresolvers import NoReverseMatch, reverse
 from django.utils.translation import ugettext_lazy as _
 
 from .models import PageMeta, TitleMeta
 
+try:
+    from django.core.urlresolvers import NoReverseMatch, reverse
+except ImportError:  # pragma: no cover
+    from django.urls import NoReverseMatch, reverse
+
```

Comment:
I'd reverse the order of these, to keep the old version as a fallback. It makes easier to remove it once we drop Django < 1.11

Reply:
thanks :+1: